app/models/presenters/unknown_tube_rack_presenter.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Presenters
+  # Presenter for unknown tube racks
+  class UnknownTubeRackPresenter < TubeRackPresenter
+    include Presenters::Statemachine::Shared
+    include Presenters::Statemachine::StateDoesNotAllowChildCreation
+    include Presenters::Statemachine::DoesNotAllowLibraryPassing
+    include Presenters::StateChangeless
+
+    validate :add_unknown_tube_rack_warnings
+
+    def robot?
+      false
+    end
+
+    def well_failing_applicable?
+      false
+    end
+
+    def add_unknown_tube_rack_warnings
+      errors.add(
+        :tube_rack,
+        "type '#{labware.purpose_name}' is not a limber tube rack. " \
+          'Perhaps you are using the wrong pipeline application?'
+      )
+    end
+
+    def label
+      Labels::PlateLabel.new(labware)
+    end
+
+    # NB. tube racks have etched barcodes and are not usually printed
+    def default_printer
+      :plate_a
```

Comment:
yeah, its a standard 'plate' format label
in reality we shouldn't need one for a tube rack as they have etched barcodes
i'm now wondering if it works, as it will expect a plate purpose not a tube rack purpose...
but maybe we should either disable it completely

Reply:
Also wondering what happens once the tube rack barcode is 'assigned' to a newer instance of that tube rack and it no longer has one